packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx

```diff
@@ -1,4 +1,4 @@
-import { LogRowModel, toDataFrame, Field, FieldCache } from '@grafana/data';
```

Comment:
In `LogRowContextProvider` we receive `sortOrder` as prop and we update the logs order in context based on it.